gammapy/background/__init__.py

```diff
@@ -10,3 +10,4 @@
 from .template import *
 from .kernel import *
 from .models import *
+from .make import *
```

Comment:
Well, most of the functions should probably move to other places. I like having the generic naming of `models.py` and `make.py`, in case other models come in the future and can share the same files. I don't have a strong opinion on that. I was just following the same convention as in the `dataset` module.

After this clarification: should I keep it as it is? or should I use `cube` specific naming?

BTW, the generic `Cube` class mentioned in https://github.com/gammapy/gammapy/pull/319#discussion-diff-36716979 could go into a `cube.py` file, similar to `maps.py`.